<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="x" default="7"/>
    <arg name="y" default="9"/>
    <arg name="theta" default="0.2"/>
    <arg name="l_tol" default="0.1"/>
    <arg name="a_tol" default="0.05"/>
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node"/>
    <node pkg="turtlesim_tutorial" type="go2pose.py" name="move_turtle" output="screen" >
        <param name="x" value="$(arg x)"/>
        <param name="y" value="$(arg y)"/>
        <param name="theta" value="$(arg theta)"/>
        <param name="l_tol" value="$(arg l_tol)"/>
        <param name="a_tol" value="$(arg a_tol)"/>
    </node>
</launch>